### DISCUSSION TITLE
Got an error when I opened Character Editor in the new update

### DISCUSSION THREAD

**Discussion by VentekOFFICIAL** [JordanSantiagoYT/FNF-JS-Engine#708] [Q&A] [answered]
![Screenshot 2024-11-01 083719](https://github.com/user-attachments/assets/c9ef34e1-111f-4693-ab5e-396a9caaca05)
Idk what it means but it popped up

**Comment by Piggymaster1222** [comment]
I did too. but could be reported in the issues section

**Reply by moxie-coder** [reply, marked as answer]
> I did too. but could be reported in the issues section

I can easily turn it into an issue

**Comment by moxie-coder** [comment]
move to the new issue I made
